contents.tex

```diff
@@ -0,0 +1,73 @@
+\mode*
+
+\section{Introduction}%
+\label{Introduction}
+
+In this lab we will evaluate some tools for private communication, in 
+particular, email and instant messaging.
+
+Email security has the longest history, this has been around since the 80s.
+Have they gained wide-spread adoption yet?
+No.
+There are two competing approaches: Pretty Good Privacy (PGP) and S/MIME 
+(comparable to HTTPS for websites).
+But there are alternatives coming, \eg ProtonMail made the headlines a few 
+years ago.
```

Comment:
Why is ProtonMail an alternative? Maybe ProtonMail with its integrated security or similar?

Reply:
Yes, they supposedly do end-to-end encryption, but it works only between ProtonMail-addresses out-of-the-box.